src/hotspot/share/opto/machnode.hpp

```diff
@@ -937,7 +937,7 @@ class MachCallJavaNode : public MachCallNode {
   bool      _optimized_virtual;      // Tells if node is a static call or an optimized virtual
   bool      _method_handle_invoke;   // Tells if the call has to preserve SP
   bool      _arg_escape;             // ArgEscape in parameter list
-  MachCallJavaNode() : MachCallNode(), _override_symbolic_info(false) {
+  MachCallJavaNode() : MachCallNode(), _override_symbolic_info(false), _arg_escape(false) {
```

Comment:
If we need to initialize one bool, then we probably need to initialize them all.  However, I think something else is going wrong to make this appear only on one platform.

Reply:
Agree, I wondered about the others too; probably we should initialize them as well.